indico/modules/designer/placeholders.py

```diff
@@ -33,15 +33,17 @@
            'RegistrationAccompanyingPersonsPlaceholder', 'RegistrationAccompanyingPersonsAbbrevPlaceholder')
 
 
-GROUP_TITLES = {
-    'registrant': _('Registrant Data'),
-    'event': _('Event Data'),
-    'fixed': _('Fixed Data')
+GROUPS = {
+    'event': {'title': _('Event Data'), 'position': 1},
+    'fixed': {'title': _('Fixed Data'), 'position': 2},
+    'registrant': {'title': _('Common Registrant Data'), 'position': 3},
+    # Come up with a better name or merge with registrant data
+    'dynamic': {'title': _('Custom Registrant Data'), 'position': 4},
```

Comment:
may be a bit wordy but something like `registrant_custom` would at least be more obvious